extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -586,47 +525,15 @@ function _jumpToMeasurement(
 
   if (enabledElement) {
     // See how the jumpToSlice() of Cornerstone3D deals with imageIdx param.
-    const viewport = enabledElement.viewport as IStackViewport | IVolumeViewport;
```

Comment:
jump to measurements for SR is broken because of this, what should we do to fix it?